### DISCUSSION TITLE
Welcome to iD Discussions!

### DISCUSSION THREAD

**Discussion by tyrasd** [openstreetmap/iD#8841] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. Here you can:

  * Find announcements of new releases, major changes and other upcoming highlights.
  * Ask questions you’re wondering about.
  * Get help with problems while working on a fork, branch or pull request.
  * Share ideas and brainstorm about potential improvements.
  * Engage with project community members and maintainers.

**Comment by tordans** [comment]
@tyrasd in nearly all repos that have both discussions and issues, I am struggling to understand what input goes where.

I suggest to add a clear guidance to a few places that helps people decide where to post stuff. And also, to introduce a strict "transfer to discussion" process that moves issues that are not really issues (defining this is tricky) to discussions.

A few places that should IMO reference this "what goes where" are …
- the repo readme
- the new-issue-chooser
- a sticky discussion entry at iD/discussions

A few ideas on a strict transfer setup …
- Define what goes where first, then move all issues that are really discussions over here (also the old ones)
- Get a few community member to help with this

**Reply by tordans** [reply]
@tyrasd as a `+1` that my concern about easy of use / understandability of discussions vs. issues is still an issue, especially for new contributors see https://github.com/openstreetmap/iD/discussions/9428#discussioncomment-4407280

Personally, I don't see iD as the place for discussion (we have the new Forum for that) and I would recommend closing this section again and continue using Issues only.
